client/layout/guided-tours/tours/site-title-tour.js

```diff
@@ -73,10 +79,9 @@ export const SiteTitleTour = makeTour(
 			shouldScrollTo
 		>
 			<Continue target="settings" step="site-title-input" click>
-				{ translate( 'Click {{strong}}{{icon/}} Settings{{/strong}} to continue.', {
+				{ translate( 'Click {{settingsButton/}} to continue.', {
```

Comment:
Hi! I've found a possible matching string that has already been translated **18** times:
`translate( 'Click {{SettingsButton/}} to continue.' )` *ES Score: 23*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).